tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -95,7 +95,7 @@ def test_that_reports_abuse_as_signed_in_user(self, mozwebqa):
 
         Assert.true(home_page.is_the_current_page)
         home_page.header.click_sign_in()
-        home_page.login(user="default")
+        home_page.login(user="new")
```

Comment:
+1 You might consider creating an api closer to what we've done on other projects, specifically creating a new user and then logging in with it -- much cleaner and more readable.